src/vast_tracker.js

```diff
@@ -333,7 +333,9 @@ export class VASTTracker extends EventEmitter {
       { isCustomCode }
     );
   }
-
+  error(macros = {}) {
+    this.trackURLs(this.ad.errorURLTemplates, { macros });
```

Comment:
trackURLs macro expect a variable containing all the macros not a object with a subobject containing the macros.
e.g if we have the macro DOMAIN='foo.bar', in your case trackURLs  will be called with param `{'macros':{'DOMAIN': 'foo.bar'}}` whereas it just expect  `{'DOMAIN':'foo.bar'}`